index.js

```diff
@@ -51,10 +56,14 @@ module.exports = function getPublicOrganisationRepositoriesFactory(token) {
 
 	var getForOrg = denodeify(github.repos.getForOrg.bind(github.repos));
 
-	return function getPublicOrganisationRepositories(organisation) {
+	return function getPublicOrganisationRepositories(organisation, privateWhitelist = []) {
```

Comment:
Could you remove the default assignment, this already happens on the line below :-)

Reply:
Oh yeah oops